modules/stardate/tests/test_stardate.c

```diff
@@ -28,7 +28,7 @@
 
 #include "cfg.h"
 
-MsgFormatOptions parse_options;
+extern MsgFormatOptions parse_options;
```

Comment:
This is actually just a workaround, by not showing the hidden dependency between this test and the `libtest/msg_parse_lib.c`. Due to the `msg_parse_lib` change, all of the unit test that uses `libtest` and the `parse_options` should include the `msg_parse_lib.h` also, and not define/declare `parse_options`.